### ISSUE TITLE
Use swipe for multi image on popover

### ISSUE THREAD

**Issue by okpierre** [jointakahe/takahe#398]
Viewing post with more than one image requires "tap" to view and then "tap" to close each time

Maybe for popup use swipes instead will improve user experience?

example from flagship instance - tap open tap close (4 times) 

![takahemultiimage](https://user-images.githubusercontent.com/1679025/212029611-8de9758c-d1fb-4d49-b5a2-f5359fd3ca9d.png)

**Comment by andrewgodwin** [comment]
Yes, keyboard navigation also does not work. I'm not going to guarantee we can fix this - we want to keep the UI relatively simple - but I'm hopeful we can.

**Comment by okpierre** [comment]
image slider

![imageslider](https://user-images.githubusercontent.com/1679025/212140719-9d025068-1243-42d5-b79f-01b1ce1fe707.gif)
